packages/expo/src/__tests__/Linking-test.ts

```diff
@@ -0,0 +1,46 @@
+import Linking from '../Linking/Linking';
+
+describe('Linking', () => {
+  describe('parse', () => {
+    test.each<[string, object]>([
```

Comment:
Is it common that expo urls have `+` in the path?

Reply:
not common i think but the code does handle that case

Reply:
What should be the behavior be when there's a `+`? Or how is the `+` currently used that it requires an additional code path?

Reply:
@ide there's still an open question here; because with the current code, urls with pluses are not handled correctly. (The test pass because there's an invalid snapshot)